assets/index.yaml

```diff
@@ -0,0 +1,229 @@
+apiVersion: v1
```

Comment:
This file should be moved to the root directory

Reply:
true true... It didn't update the root `index.yaml` 🤦